cli/src/main.rs

```diff
@@ -308,6 +312,12 @@ fn main() {
             let output = result.ok_or_abort_with("Failed to show Solido data.");
             print_output(output_mode, &output);
         }
+        SubCommand::ShowAuthorities(solido_pubkey) => {
+            let result =
+                config.with_snapshot(|_config| command_show_solido_authorities(&solido_pubkey));
+            let output = result.ok_or_abort_with("Failed to show Solido data.");
```

Comment:
Nit:  Shouldn't this be a little more specific to the sub command, i.e. "Failed to show authorities for Solido instance"

Reply:
Thanks! That was copy-pasta mistake. Changed it.